pkg/standalone/standalone.go

```diff
@@ -1062,3 +1066,21 @@ func getPlacementImageWithTag(name, version string) string {
 	}
 	return fmt.Sprintf("%s:%s", name, version)
 }
+
+func resolveImageURI(imageInfo daprImageInfo) (string, error) {
+	if imageInfo.imageRegistryURL != "" {
+		if imageInfo.imageRegistryURL == ghcrURI || imageInfo.imageRegistryURL == "docker.io" {
+			err := fmt.Errorf("flag %s not set correctly", "--image-registry")
+			return "", err
+		}
+		return fmt.Sprintf(privateRegTemplateString, imageInfo.imageRegistryURL, imageInfo.ghcrImageName), nil
```

Comment:
Should this be `imageInfo.ghcrImageName` is `imageRegistryURL` is set to Docker?

Reply:
@yaron2 here `imageRegistryURL` will be provided by user. It is for user who have images in their private registry. 
Also, in case of private registries we decided to keep images under similar directory structure as it for ghcr.

example - 
private registry can be - `docker.io/pravin` or any other value like `ghcr.io/<pravin|some_username>`. User can not just have `docker.io` or `ghcr.io` or `ghcr.io/dapr`. Last one is to avoid conflict with existing dapr `Github container registry`.
and expectation is, this user will have images under his registry like below - 
dapr/dapr:<version>
dapr/3rdparty/zipkin
dapr/3rdparty/rejon